ui-lib/components/Button/index.js

```diff
@@ -0,0 +1,170 @@
+// @flow
+import React, { useState } from "react";
+import styled, { css } from "styled-components";
+import { fontSize, color } from "styled-system";
+import type { ThemedComponent } from "@ui/styles/StyleProvider";
+import fontFamily from "@ui/styles/styled/fontFamily";
+import { fontSizes } from "@ui/styles/theme";
+import ChevronDown from "@ui/icons/ChevronDown";
+
+type Props = {
+  Icon?: React$ComponentType<*>,
+  children?: React$Node,
+  onClick: () => void,
+  ff?: string,
+  color?: string,
+  fontSize?: number,
+  type?: "primary" | "secondary",
+  iconPosition?: "right" | "left",
+};
+
+const IconContainer: ThemedComponent<{ iconPosition: "right" | "left" }> = styled.div`
+  display: inline-block;
+  ${p => (p.iconPosition === "left" ? `margin-right: 10px;` : `margin-left: 10px;`)}
+  padding-top: 0.2em;
+`;
+
+export const Base: ThemedComponent<{
+  Icon?: React$ComponentType<*>,
+  ff?: string,
+  color?: string,
+  fontSize?: number,
+  type?: "primary" | "secondary",
+  iconPosition?: "right" | "left",
+  ...
```

Comment:
This is an artefact from a conflict, isn't it?

Reply:
its a way say to flow it can accept additional props :sweat_smile: